azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/ContainerServiceMasterProfileCount.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.compute;
+
+/**
+ * Defines values for ContainerServiceOchestratorTypes.
+ */
+public enum ContainerServiceMasterProfileCount {
+    /** Enum value Min. */
```

Comment:
a better description of the enum value is needed.  Something like 'The minimum valid number of master nodes.`
etc. etc for the others...